### PR TITLE
auto merge for Glacier peak

### PR DESCRIPTION
**Pull request by noam987**
Please Review this to ensure that nothing related to Teleop was accidentally changed.  
The removal of the arm pid is intentional as it is unused anyways

### CODE REVIEW

File: code/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -172,6 +174,9 @@ public void robotPeriodic() {
   @Override
   public void disabledInit() {
     _robot.disabledInit();
+    new Limelight(_config).setLEDState(1);
```
Comment (thetaspirit) [inline code/src/main/java/frc/robot/Robot.java]: Move this to robotinit

File: code/src/main/java/frc/robot/RobotConfig.java [hunk 2]
```diff
@@ -131,10 +131,10 @@
     }
 
     public final class Limelight {
-        public final double kP = .01;
```
Comment (thetaspirit) [inline code/src/main/java/frc/robot/RobotConfig.java:134]: Make sure these changes to pid are what you want, even for teleop

File: code/src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -38,30 +56,71 @@
...
     climb.setDefaultCommand(climbCommand);
 
+    m_autonomousCommand = new SequentialCommandGroup(new ShooterStartupCommand(logger, shooter),
+        new TimedAutonMoveBackward(logger, _config, drivetrain).withTime(1.5).withVelocity(1),
+        new TimedAutonMoveBackward(logger, _config, drivetrain).withTime(1).withVelocity(.5).withTimeout(3),
+        new TrackVisionTarget(logger, config, drivetrain, new Limelight(config)).withTimeout(4),
+        new ShootOneBallCommand(logger, shooter), new ShootOneBallCommand(logger, shooter),
+        new ShootOneBallCommand(logger, shooter), new ShooterShutdownCommand(logger, shooter));
+
+    // TODO finish this auton after Glacier Peak.
```
Comment (thetaspirit) [inline code/src/main/java/frc/robot/RobotContainer.java:83]: Remove this for when you merge to master, keep it in the feature branch.   Keep the Todo stub tho

File: code/src/main/java/frc/robot/commands/navcommands/TrackVisionTarget.java [hunk 4]
```diff
@@ -38,6 +38,7 @@ public TrackVisionTarget(final ILogger logger, RobotConfig config, IDrivetrainSu
     @Override
     public void initialize() {
         super.initialize();
+
```
Comment (thetaspirit) [inline code/src/main/java/frc/robot/commands/navcommands/TrackVisionTarget.java]: Remove empty lines added

**Comment by noam987** [comment]
@delaynieym as the driver, what teleop commit do you want into master. Auton is the same for most of them, with the acception being the added move forward auton.

**Review by thetaspirit — APPROVED** [review]

**Pull request merged**